app/javascript/chart_builder.js

```diff
@@ -48,13 +48,40 @@ document.addEventListener('turbolinks:load', function() {
     }
     throw new Error('Request for run from API failed')
   }).then(function(bodies) {
-    const runs = bodies.map(body => body.run)
+    let runs = bodies.map(body => body.run)
     const timing = new URLSearchParams(window.location.search).get('timing') || runs[0].default_timing
 
     document.getElementById('chart-spinner').hidden = true
     document.getElementById('chart-holder').hidden = false
     if (runs[0].histories.length !== 0) {
       const skipped = `${timing}time_skipped`
+      const duration = `${timing}time_duration_ms`
+      runs = runs.map((run) => (
+        {
+          ...run,
```

Comment:
More syntax I didn't know about! This is so much better than `_.merge` or whatever I was using.